minimal/staging.yml

```diff
@@ -53,4 +53,13 @@ services:
     volumes_from:
       - {{ project_name }}_web
     ports:
-      - "0.0.0.0:8005:80"
\ No newline at end of file
+      - "0.0.0.0:8005:80"
+
+  {{ project_name }}_certbot:
+    restart: always
+    build: ./compose/certbot/production
```

Comment:
@pslobo i just noticed, i think this build is wrong because this path does not exist. your current dockerfile is in `minimal/compose/certbot/Dockerfile` so it should be `./compose/certbot/staging/Dockerfile`.

Also, we should probably separate the dockerfiles for staging and production and use it accordingly since they should be different containers and have different certificates. so we need a `./compose/certbot/staging/Dockerfile` and a `./compose/certbot/production/Dockerfile`